src/pe_source/data/sixgill/__init__.py

```diff
@@ -0,0 +1 @@
+"""Initialize Cybersixgill folder."""
```

Comment:
If this directory is meant to be a Python package, then the contents of this file should be similar to those of https://github.com/cisagov/skeleton-python-library/blob/develop/src/example/__init__.py. Otherwise, this file is not needed.

Reply:
File removed here: https://github.com/cisagov/pe-reports/pull/117/commits/3eb78d8b7cc157a54be3f17b2c48340f2b7f576e